test/e2e/util/util.go

```diff
@@ -195,6 +201,100 @@ func WaitForGuestKubeConfig(t *testing.T, ctx context.Context, client crclient.C
 	return data, nil
 }
 
+func WaitForKubeVirtMachines(t *testing.T, ctx context.Context, client crclient.Client, hostedCluster *hyperv1.HostedCluster, count int32) {
+	g := NewWithT(t)
+	start := time.Now()
+
+	localCtx, cancel := context.WithTimeout(ctx, 10*time.Minute)
+	defer cancel()
+	t.Logf("Waiting for %d kubevirt machines to come online", count)
+
+	namespace := manifests.HostedControlPlaneNamespace(hostedCluster.Namespace, hostedCluster.Name).Name
+
+	err := wait.PollUntil(5*time.Second, func() (done bool, err error) {
+		var kvMachineList capikubevirt.KubevirtMachineList
+
+		err = client.List(ctx, &kvMachineList, crclient.InNamespace(namespace))
+		if err != nil {
+			t.Errorf("Failed to list KubeVirtMachines: %v", err)
+			return false, nil
+		}
+
+		readyCount := 0
+		for _, machine := range kvMachineList.Items {
+			if machine.Status.Ready {
+				readyCount++
+			}
+		}
+		if int32(readyCount) < count {
+			return false, nil
+		}
+
+		return true, nil
+	}, localCtx.Done())
+	g.Expect(err).NotTo(HaveOccurred(), "timeout waiting for kubevirt machines to become ready")
+
+	t.Logf("KubeVirtMachines are ready in %s", time.Since(start).Round(time.Second))
+}
+
+func WaitForKubeVirtCluster(t *testing.T, ctx context.Context, client crclient.Client, hostedCluster *hyperv1.HostedCluster) {
```

Comment:
can you please move this into a kubevirt file?

Reply:
yep, no problem